app/javascript/components/Modal.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from "react";
```

Comment:
Lets put this and style in their own folder ie. 

`app/javascript/components/modal/Modal.tsx`